docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,29 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing registration password from OS native keychain
+
+=== Windows
+
+In windows the password is stored using *Credential Manager*.
+To delete the stored password run the following command in command prompt:
+
+----
+CMD> cmdkey /delete:minishift:<username>
+----
+
+=== GNU/Linux
+
+In linux the password is stored using *gnome-keyring*.
+To delete the stored password run the following command:
+
+----
+$ secret-tool clear service minishift username <username>
+----
+
+=== MacOS
+
+For macOS use the *Keychain Access* GUI app to delete the stored password, look for an entry  under passwords that says minishift.
```

Comment:
Unfortunately, I don't use macOS, so I can't see the GUI being described here myself. What we could use, though, is a little bit more instruction about what to do with the entry named `minishift`. I'd also recommend rewording this some, and I'd make referenced GUI elements bold; something like this:

```adoc
For macOS, use the *Keychain Access* application to delete the stored password.
Look for an entry under *Passwords* named `minishift`.
// Describe further actions needed here.
```

Reply:
On a second thought i should just mention the command to delete it like the rest